11/umbraco-forms/editor/creating-a-form/creating-a-contact-us-form.md

```diff
@@ -0,0 +1,265 @@
+# Creating a Contact Us Form
+
+In this article, we'll look at creating a Contact Us Form.
+
+## Video Tutorial
+
+{% embed url="https://www.youtube.com/watch?ab_channel=UmbracoLearningBase&v=TWLqt-jVdyQ" %}
+How to use Language Variants in Umbraco
+{% endembed %}
+
+## Creating a Composition
+
+To create a Composition, follow these steps:
+
+1. Go to **Settings**.
+2. Expand the **Document Types** folder in the **Settings** tree.
+3. Select **...** next to the **Compositions** folder.
+4. Click **Composition**.  
+
+    ![Creating a Composition](images/creating-a-composition.png)
+5. Enter a **Name** for the **Composition**. Let's call it _Title Box_.
+6. Add the following fields with the respective specifications:
+
+    | Group     | Field Name | Alias    | Data Type  |
+    |-----------|------------|----------|------------|
+    | Title Box | Title      | title    | Textstring |
+    | Title Box | Subtitle   | subtitle | Textarea   |
+
+    ![Add Composition Properties](images/composition-properties.png)
```

Comment:
It's OK to not make the images/tables a part of the list - the ordered list will continue under it.